### PR TITLE
A bunch of changes

### PR DESCRIPTION
**Pull request by alazymeme**
- Fixed up and formatting capitalization in the readme
- Added a favicon
- Made the light/slate theme change as per preference
- Made the wiki act like a single page (not having to reload everytime you want to change pages)
- Added the github repo + changed the icon + removed the edit button
- Added a revision date to the docs (localized)
- Added discord, github and donate social links
- Updated build workflow to install the revision date plugin

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -2,5 +2,28 @@ site_name: Chatterino
 theme:
   name: material
   logo: https://chatterino.com/img/icon-256.png
+  favicon: https://chatterino.com/img/icon-256.png
+  palette:
+    scheme: preference
+  features:
+    - instant
+  icon:
+    repo: fontawesome/brands/github
```
Comment (fourtf) [inline mkdocs.yml:11]: Do we need to put a license notice somewhere?

File: mkdocs.yml [hunk 2]
```diff
@@ -2,5 +2,28 @@ site_name: Chatterino
 theme:
   name: material
   logo: https://chatterino.com/img/icon-256.png
+  favicon: https://chatterino.com/img/icon-256.png
+  palette:
+    scheme: preference
+  features:
+    - instant
```
[the remark this answers is not in the document]
Comment (alazymeme) [reply mkdocs.yml:9]: `scheme: preference` will change to light/dark mode depending on what the user has set: https://squidfunk.github.io/mkdocs-material/setup/changing-the-colors/#color-scheme
instant loading allows you to load pages without fully reloading the page: https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#instant-loading

File: mkdocs.yml [hunk 3]
```diff
@@ -2,5 +2,28 @@ site_name: Chatterino
...
+  icon:
+    repo: fontawesome/brands/github
 plugins:
   - search
+  - git-revision-date-localized:
+      type: datetime
+      fallback_to_build_date: true
+repo_url: https://github.com/chatterino/chatterino2
```
Comment (fourtf) [inline mkdocs.yml:17]: This linking to the chatterino repo instead of the wiki seems unintuitive

File: mkdocs.yml (hunk 1 above)
Comment (alazymeme) [reply mkdocs.yml:11]: Don't think so, these are bundled in with the theme and are free to use. https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#repository-icon
https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/source.html

File: mkdocs.yml (hunk 3 above)
Comment (alazymeme) [reply mkdocs.yml:17]: Entirely up to you. The wiki is about chatterino2, so I thought it'd be most appropriate to link to the chatterino2 repo. FYR I also have this linked in the social icons at the bottom of the page.

Comment (alazymeme) [reply mkdocs.yml:17]: And I've removed the edit icons (they didn't look good to me), I can have them re-added if you want

**Pull request merged**